src/async-wrap.cc

```diff
@@ -407,6 +407,40 @@ static void DisablePromiseHook(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+class AsyncWrap::DestroyParam {
+ public:
+  double asyncId;
+  v8::Persistent<Function> callback;
+};
+void AsyncWrap::WeakCallback(
```

Comment:
Can you leave two lines of space between definitions? Otherwise this is very dense code :)